File: FaceSample/app/src/main/java/com/regula/facesample/items/features/FaceSdkInitItem.java
package com.regula.facesample.items.features;

import android.content.Context;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesdk.FaceSDK;

public class FaceSdkInitItem extends CategoryItem {

   @Override
   public void onItemSelected(Context context) {
      FaceSDK.Instance().init(context, (status, exception) -> { });
   }

   @Override
   public String getDescription() {
      return "Initialization FaceSDK resources";
   }

   @Override
   public String getTitle() {
      return "Initialization SDK";
   }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/features/FaceSdkInitItem.kt
package com.regula.facesamplekotlin.items.features

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesdk.FaceSDK
import com.regula.facesdk.exception.InitException

class FaceSdkInitItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        FaceSDK.Instance().init(
            context
        ) { status: Boolean, exception: InitException? -> }
    }

    override val title: String
        get() = "Initialization SDK"

    override val description: String
        get() = "Initialization FaceSDK resources"
}